src/Geometry.ts

```diff
@@ -241,12 +241,12 @@ function computeBottomGeometry({ displayArea, fromRect, requestedContentSize, ar
   let minX = displayArea.x;
   let preferedX = fromRect.x + (fromRect.width - viewWidth) / 2;
 
-  var popoverOrigin = new Point(
+  const popoverOrigin = new Point(
     Math.min(maxX, Math.max(minX, preferedX)),
     preferedY
   );
 
-  var anchorPoint = new Point(fromRect.x + fromRect.width / 2.0, fromRect.y + fromRect.height);
+  let anchorPoint = new Point(fromRect.x + fromRect.width / 2.0, fromRect.y + fromRect.height);
```

Comment:
No big deal, just wanted to let you know for your knowledge that this could be a const as well, you can change the properties of a const object as long as you don't re-assign the variable itself.